pkg/runtime/podman/podman.go

```diff
@@ -1,4 +1,6 @@
-package runtime
+// +build !darwin
+
+package podman
```

Comment:
I believe we should do the same for podman: podman_linux.go & podman_darwin.go

Reply:
:+1: added `podman_linux`